package.json

```diff
@@ -48,7 +48,8 @@
     "semver": "^7.3.8",
     "strftime": "^0.10.1",
     "stylelint-config-standard": "^31.0.0",
-    "swr": "^2.1.1"
+    "swr": "^2.1.1",
+    "framer-motion": "10.12.4"
```

Comment:
Hmm, did you use npm install to add this dependency?

Because it's out of order here, it should probably be after the `feed` dependency. And the version string should have a `^` on it 🤔